packages/mui-base/package.json

```diff
@@ -43,7 +43,7 @@
     "@babel/runtime": "^7.24.7",
     "@floating-ui/react-dom": "^2.1.0",
     "@mui/types": "workspace:^",
-    "@mui/utils": "workspace:^",
+    "@mui/utils": "workspace:",
```

Comment:
It should be `workspace:*` to reference the exact version (see https://pnpm.io/workspaces)

Reply:
Thanks